src/cmd_line/commands/file.ts

```diff
@@ -8,16 +8,16 @@ import { Message } from './../../util/message';
 const untildify = require('untildify');
 
 export enum FilePosition {
-  CurrentWindow,
-  NewWindow,
+  NewWindowVerticalSplit,
+  NewWindowHorizontalSplit,
 }
 
 export interface IFileCommandArguments extends node.ICommandArgs {
   name: string | undefined;
   bang?: boolean;
   position?: FilePosition;
   lineNumber?: number;
-  createFileIfNotExists?: boolean;
+  new?: boolean; // Create a new untitled file
```

Comment:
I prefer the old variable name as it was descriptive and wouldn't require a preceding comment.

Reply:
Faire enough.